### DISCUSSION TITLE
Laravel livewire data binding always get zero value from input fields

### DISCUSSION THREAD

**Discussion by GdcJanaka** [livewire/livewire#4056] [1. Help]
### Controller
![image](https://user-images.githubusercontent.com/55373578/138055883-98b85231-3204-4cc8-afa2-abda3a8e64b0.png)

### Blade
![image](https://user-images.githubusercontent.com/55373578/138056026-25ebddc0-5c44-4658-9332-6e6d3250895b.png)

### Main Blade
![image](https://user-images.githubusercontent.com/55373578/138056163-8faa1727-11c9-49b9-8e8d-32865edbb714.png)

### Result
![image](https://user-images.githubusercontent.com/55373578/138055077-61c0bfce-873c-44f2-b54e-b233135056dc.png)

Php Version 8.0.11
Laravel Version 8.65.0
Livewire Version 2.7.1

**Comment by GdcJanaka** [comment]
Its working on Operamini web browser, but not on Chrome

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by joshhanley** [comment]
@GdcJanaka when posting code, can you please make sure to paste actual code and not text, so we can can copy and test it.

When you paste code into github, you need to surround it in triple back ticks and then specify the file type like php or blade.

For example
\```php
\\ PHP code here
\```

\```blade
\\ HTML and blade code here
\```

But I have tested this in Chrome and it is all working for me. Maybe make sure that Livewire is running properly in your browser console by typing `Livewire` and checking that a Livewire object is returned.

Hope this helps!

**Comment by egovaflavia** [comment]
I've same problem, is there any solution ?
